### PR TITLE
add a new doc: 10 minutes guide

### PR DESCRIPTION
**Pull request by en-jin19**
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

### CODE REVIEW

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+```
+NAME                           READY   STATUS    RESTARTS   AGE
+mysql-query-6dd8bd5474-5zbkw   1/1     Running   0          60s
+mysql-query-6dd8bd5474-k8bgs    1/1     Running   0          60s
+```
+
+In the result, you can find that there are two Pods that share the query request, and the mysql-query service has configured `NodePort`. Therefore, the Pods can directly access the service on the nodes of the Kubernetes cluster through the NodePort (`30001` in here).
+
+4. Run the following command to access kind-control-planeand send HTTP requests:
```
Comment (WangXiangUSTC) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:156]:
```suggestion
4. Run the following command to access kind-control-plane and send HTTP requests:
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
+---
+title: 10 Minute Guide to Chaos Mesh v2.1
+---
+
+Chaos Mesh v2.1.0 introduces the following significant changes, which expand the user scenarios of Chaos Mesh and improves overall usability:
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:5]:
```suggestion
Chaos Mesh v2.1.0 introduces the following significant changes, which expand the user scenarios of Chaos Mesh and improve overall usability:
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+- Support creating and running Chaos experiments in a physical machine, an important step for Chaos Mesh to become a platform.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:7]:
```suggestion
- Support creating and running Chaos experiments in a physical machine, an important step for Chaos Mesh to become a Chaos Engineering platform.
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+- Optimize the workflow to display the workflow-related events and configurations and to create HTTP tasks in the workflow, which greatly adds the user scenarios of Workflow.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:8]:
```suggestion
- Optimize the workflow to display the workflow-related events and configurations and to create HTTP tasks in the workflow, which greatly extends the user scenarios of Workflow.
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+- Support creating HTTPChaos on Chaos Dashboard, which improves the usability of HTTPChaos.
+- Supports simulating the faults of JVM application through [Byteman](https://byteman.jboss.org), which improves the usability and scalability of JVMChaos.
+- Integrates GCP certification in Chaos Dashboard.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:11]:
```suggestion
- Integrates GCP certification into Chaos Dashboard.
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+:::
+
+1. Clone Chaos Mesh project
+
+You can clone [Chaos Mesh project](https://github.com/chaos-mesh/chaos-mesh) on Github by running the following command:
+
+```
+git clone https://github.com/chaos-mesh/chaos-mesh.git && \
+cd chaos-mesh/
+```
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:39]:
```suggestion
    You can clone [Chaos Mesh project](https://github.com/chaos-mesh/chaos-mesh) on Github by running the following command:

    ```
    git clone https://github.com/chaos-mesh/chaos-mesh.git && \
    cd chaos-mesh/
    ```
```

Comment (ran-huang) [reply docs/10-minute-guide-to-chaos-mesh-v21.md:39]: Please add indentation for other items under a list.

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+## Prepare the environment
+
+### Deploy Kubernetes and Chaos Mesh
+
+Before you begin, make sure you have installed [Git](https://git-scm.com) and [Docker](https://www.docker.com) on your computer.
+
+In this section, you are going to deploy Kubernetes and Chaos Mesh as follows:
+
+- Deploy a Kubernetes cluster using an installation script from the Chaos Mesh project.
+- Install Chaos Mesh in that Kubernetes cluster.
```
Comment (WangXiangUSTC) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:24]: the installation script will do these two things.

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+
+:::
+
+1. Clone Chaos Mesh project
+
+You can clone [Chaos Mesh project](https://github.com/chaos-mesh/chaos-mesh) on Github by running the following command:
+
+```
+git clone https://github.com/chaos-mesh/chaos-mesh.git && \
+cd chaos-mesh/
```
Comment (WangXiangUSTC) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:38]: I forget to checkout to the v2.1.0, please add a command after L38: `git checkout v2.1.0`

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 9]
```diff
@@ -0,0 +1,298 @@
...
+
+To visit Chaos Dashboard in the browser, you need to run the following command to forward the port for the Chaos Dashboard service:
+
+```
+kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333 --address 0.0.0.0
+```
+
+After running the command, you can access Chaos Dashboard by visiting <127.0.0.1:2333> in the browser. The Dashboard interface is as follows:
+
+[img/quick-start-chaos-mesh-dashboard.png]
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:61]: Please fix images in this doc.

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 10]
```diff
@@ -0,0 +1,298 @@
...
+cd chaosd-v1.1.0-linux-amd64
+```
+
+3. Enable Chaosd in the service mode:
+
+```
+nohup chaosd server --port 31767 2>&1 chaosd.log &
+```
+
+After that, you can send the request for Chaos experiments to Chaosd through the port “31767” of your local machine.
```
Comment (WangXiangUSTC) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:86]:
```suggestion
After that, you can send the request for Chaos experiments to Chaosd through the port "31767" of your local machine.
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 11]
```diff
@@ -0,0 +1,298 @@
...
+
+#### Deploy mysql-query service
+
+[mysql-query](https://github.com/WangXiangUSTC/byteman-example/tree/chaos_demo/mysqldemo) is a demo application written in Java, whose image is uploaded to Docker Hub. To deploy the service in the Kubernetes cluster, take the following steps:
+
+1. Download the [`deployment.yaml`](https://github.com/WangXiangUSTC/byteman-example/blob/chaos_demo/mysqldemo/deployment.yaml) file and modify the configuration.
+
+:::note
+
+You need to replace the IP address in the environment variable `MYSQL_DSN` with the address of the server where  MySQL is installed.
```
Comment (WangXiangUSTC) [inline docs/10-minute-guide-to-chaos-mesh-v21.md:129]:
```suggestion
You need to replace the IP address in the environment variable `MYSQL_DSN` with the address of the server where MySQL is installed.
```

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

File: docs/10-minute-guide-to-chaos-mesh-v21.md (hunk 7 above)
Comment (en-jin19) [reply docs/10-minute-guide-to-chaos-mesh-v21.md:24]: > the installation script will do these two things.

have updated, PTAL, thanks!

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 12]
```diff
@@ -0,0 +1,299 @@
...
+    ```
+
+3. Run the following SQL statements to create a user that can access the database externally, and grant all privileges to the user:
+
+    ```
+    CREATE USER 'chaos'@'%' IDENTIFIED BY 'chaos-mesh';
+    GRANT ALL PRIVILEGES ON * . * TO 'chaos'@'%';
+    ```
+
+4. Prepare the dataset for testing. You can run  [example.SQL](https://github.com/WangXiangUSTC/byteman-example/blob/chaos_demo/mysqldemo/example.SQL) in MySQL to generate the test data.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md]:
```suggestion
4. Prepare the dataset for testing. You can run [`example.SQL`](https://github.com/WangXiangUSTC/byteman-example/blob/chaos_demo/mysqldemo/example.SQL) in MySQL to generate the test data.
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 13]
```diff
@@ -0,0 +1,299 @@
...
+
+![Two mysql-query Pods](./img/quick-start-two-pods-for-sql-queries.png)
+
+If one of the Pods fails and cannot be connected, a highly available system can automatically forward the requests to another Pod. You can create a Chaos experiment to verify the service's high availability. The steps are as follows:
+
+1. Click **New experiment** in Chaos Dashboard:
+
+    ![New experiment](./img/quick-start-new-experiment-dashboard.png)
+
+2. Select **Kubernetes** as the experiment environment and ** HTTP Fault** as the experiment type:
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md]:
```suggestion
2. Select **Kubernetes** as the experiment environment and **HTTP Fault** as the experiment type:
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 14]
```diff
@@ -0,0 +1,299 @@
...
+    docker exec -it kind-control-plane curl -X GET "http://127.0.0.1:30001/query?sql=SELECT%20*%20FROM%20chaos.website"
+    ```
+
+    You can see that the command sometimes returns a normal response, but sometimes reports an error:
+
+    ![Report errors](./img/quick-start-report-error.png)
+
+    The experiment shows that this service is not highly available. You have successfully found a fault in the service architecture through a HTTP fault injection.
+
+    After the experiment, you might need to consider how to optimize the service architecture, such as using Ingress and NGINX.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md]: L206 - L218 这一段应该不需要缩进哈

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 15]
```diff
@@ -0,0 +1,299 @@
...
+
+### Experiment 2: What if the query returns an exception?
+
+Even though all of your SQL queries executed in normal situations return normal data, that does not mean your code has no bugs. If the query returns an exception, you might wonder whether your application can handle the exception and return the error message. In such cases, you can verify that situation by running a Chaos experiment.
+
+Before creating the experiment, you need to understand the code logic of your application. The [querySQL](https://github.com/WangXiangUSTC/byteman-example/blob/318a35fbff3e1648464b69bf892fcc0ec1860a28/mysqldemo/src/main/java/com/mysqldemo/App.java#L126) function executes SQL statements and can throw a `java.sql.SQLException` exception. You can inject faults into this function.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md]:
```suggestion
Before creating the experiment, you need to understand the code logic of your application. The [`querySQL`](https://github.com/WangXiangUSTC/byteman-example/blob/318a35fbff3e1648464b69bf892fcc0ec1860a28/mysqldemo/src/main/java/com/mysqldemo/App.java#L126) function executes SQL statements and can throw a `java.sql.SQLException` exception. You can inject faults into this function.
```

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 16]
```diff
@@ -0,0 +1,299 @@
...
+    ```
+    docker exec -it kind-control-plane curl -X GET "http://127.0.0.1:30001/query?sql=SELECT%20*%20FROM%20chaos.test"
+    curl: (18) transfer closed with outstanding read data remaining
+    ```
+
+    The application reports an error, but the result is not the exception information of the application. The logs of the two mysql-query Pods do not contain any notable information.
+
+    To find out the root cause, you need to read the code and find out which part of the code is executed after `java.sql.SQLException` is thrown. Through investigation, you can find an exception caused by a null pointer.
+
+    You have successfully found a fault in the code through a JVM fault injection.
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md]: L242 - L253 不用缩进

File: docs/10-minute-guide-to-chaos-mesh-v21.md [hunk 17]
```diff
@@ -0,0 +1,299 @@
...
+    Elapsed time: 39(ms)
+    ```
+
+    You can see that the query successfully returns a result. The elapsed time does not increase significantly.
+
+    For now, you do not have to worry about the performance of MySQL, and you can even deploy other applications on this physical machine. However, when the data volume is large and the database must serve thousands of queries per second, MySQL might not be able to provide your application with acceptable performance. You might consider try TiDB at this time.
+
+    By going through this tutorial, you have now tried out three experiments of Chaos Mesh v2.1 (HTTP, JVM, physical machine environment) and felt the charm of Chaos Mesh and Chaos Engineering.
+
+    If you have any questions when you use Chaos Mesh, feel free to [tell us](https://github.com/chaos-mesh/chaos-mesh/issues).
```
Comment (ran-huang) [inline docs/10-minute-guide-to-chaos-mesh-v21.md]: L277 - L299 不用缩进

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**